src/pages/careers/_who-we-look-for.js

```diff
@@ -81,21 +81,11 @@ const ImageWrapper = styled.div`
 `
 const query = graphql`
     query {
-        teamfocus: file(relativePath: { eq: "careers/team-focus.png" }) {
-            childImageSharp {
-                fluid(maxWidth: 650, srcSetBreakpoints: [400, 600, 960, 1280, 1920]) {
-                    ...GatsbyImageSharpFluid_withWebp_noBase64
-                    originalName
-                }
-            }
+        teamfocus: file(relativePath: { eq: "careers/team-focus.jpg" }) {
+            ...backGroundBlur
```

Comment:
why change this?

Reply:
previously qa raised that this image is blur. we have a separate card for it already, reverting this 🙏